src/domElements.js

```diff
@@ -0,0 +1,93 @@
+/* global document */
+const mainContent = document.querySelector('#content');
+const selectProject = document.querySelector('#mySelectProject');
+const addProjectBtn = document.querySelector('#addProjectBtn');
+const toDoBtn = document.querySelector('#toDoBtn');
+
+const projectForm = document.querySelector('#myProjectForm');
+const projectName = document.querySelector('#projectName');
+const createProject = document.querySelector('#createProject');
+const cancelProject = document.querySelector('#cancelProject');
+
+const myTodoForm = document.querySelector('#myTodoForm');
+const todoTitle = document.querySelector('#todoTitle');
+const todoDescription = document.querySelector('#todoDescription');
+const todoDueDate = document.querySelector('#todoDueDate');
+const todoPriority = document.querySelector('#todoPriority');
+
+// const deleteTodo = document.getElementsByClassName('deleteBtn');
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history.